manifests/k/KDE/ruqola/1.6.1/KDE.ruqola.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: KDE.ruqola
+PackageVersion: 1.6.1
+PackageLocale: en-US
+Publisher: KDE

```

This needs to match ARP
```suggestion
Publisher: KDE e.V.
```

---

Done.